common/types/socket/types.ts

```diff
@@ -16,7 +16,13 @@ export interface EmitTimerResponseArgs {
 
 // all emit events
 export interface ServerToClientEvents {
-	usersInRoom: (data: number) => void;
+	usersInRoom: ({
+		numUsers,
+		userList,
+	}: {
+		numUsers: number;
+		userList: string[];
+	}) => void;
```

Comment:
Rather than defining the argument types inline, should we create another interface for this and keep it consistent with the other methods? For example:

```typescript
export interface EmitUsersInRoomArgs {
    numUsers: number;
    userList: string[];
}

export interface ServerToClientEvents {
    usersInRoom: (data: EmitUsersInRoomArgs) => void;
    // ...
}
```